0.Preprocess_Data/1.preprocessing_morphology_data.ipynb

```diff
@@ -898,7 +898,7 @@
                     "name": "stderr",
```

Comment:
<p>May also be able to simplify this with a for loop</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>yes, I went ahead and replaced with a for loop</p>